typescript/helloworld/contracts/HelloWorld.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
This file and all files under `/helloworld/src/` are unchanged. Recommend just reviewing the config files